coffee_time/coffee.py

```diff
@@ -0,0 +1,20 @@
+import math
+
+
+class Coffee:
+    """
+    This is a new_coffee class.
+    """
+
+    def __init__(self, coffee_spoons, sugar, water):
+        self.coffee_spoons = coffee_spoons
+        self.sugar = sugar
+        self.water = water
+
+    def prepare_coffee(self):
+        coffee_quantity = 3*self.coffee_spoons + 2*self.sugar + 5*self.water
+        return coffee_quantity
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water
```

Comment:
pep8 recommends to leave spaces between operators.
https://www.python.org/dev/peps/pep-0008/#other-recommendations